src/components/Dashboard/DashboardCard/DashboardCardDropdownBody.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const DashboardCardDropdownBody = ({ children }) => <div className="kubevirt-dropdown__items">{children}</div>;
```

Comment:
I'd rather name this more generic, like `DashboardCardActionsBody`

Reply:
ack